api/cas-server-core-api-mfa/src/main/java/org/apereo/cas/authentication/MultifactorAuthenticationFailureMode.java

```diff
@@ -0,0 +1,33 @@
+package org.apereo.cas.authentication;
+
+import org.apereo.cas.services.RegisteredService;
+import org.apereo.cas.services.RegisteredServiceMultifactorPolicy;
+import org.springframework.core.Ordered;
+
+import java.io.Serializable;
+
+/**
+ * This is {@link MultifactorAuthenticationFailureMode}.
+ *
+ * @author Travis Schmidt
+ * @since 6.0.5
+ */
+@FunctionalInterface
+public interface MultifactorAuthenticationFailureMode extends Serializable, Ordered {
```

Comment:
If you're going to evaluate something, the component would need to be called `XYZ-Evaluator`.

Reply:
So should MultifactorAuthenticationProviderBypass be renamed to MultifactorAuthenticationProviderBypassEvaluator as well?

Reply:
That would be a good change sure, but probably should be done when this ge sported over to master, as an API change is disallowed here.